test/qtismtest/runtime/expressions/operators/PatternMatchProcessorTest.php

```diff
@@ -110,7 +110,9 @@ public function patternMatchProvider()
             [new QtiString('string'), 'shell', false],
             [new QtiString('stringString'), '.*', true], // in xml schema 2, dot matches white-spaces
```

Comment:
This test doesn't actually test what the comment says. The searched string should contain at least one space:
```
[new QtiString('string String'), '.*', true], // in xml schema 2, dot matches white-spaces
```